DESCRIPTION

```diff
@@ -22,13 +22,13 @@ Roxygen: list(markdown = TRUE, r6 = FALSE)
 SystemRequirements: CmdStan (https://mc-stan.org/users/interfaces/cmdstan)
 Imports: 
     checkmate,
+    jsonlite (>= 1.2.0),
```

Comment:
@rok-cesnovar At this point we use jsonlite in enough places that we might as well move it from Suggests to Imports.

Reply:
I agree.